app/src/main/java/com/example/taskwise_eventmaster/presentation/events_screen/EventCardDetails.kt

```diff
@@ -0,0 +1,135 @@
+package com.example.taskwise_eventmaster.presentation.events_screen
+
```

Comment:
There are a lot of identical or similar composables in `EventsScreen` and `EventCardDetails`. Consider extracting and reusing them.